io.openems.edge.ess.sma/src/io/openems/edge/sma/SunnyIsland6Ess.java

```diff
@@ -108,20 +108,20 @@ public void applyPower(int activePowerL1, int reactivePowerL1, int activePowerL2
 	@Override
 	protected ModbusProtocol defineModbusProtocol() {
 		ModbusProtocol protocol = new ModbusProtocol(this, //
-				new FC3ReadRegistersTask(30051, Priority.ONCE, //
-						m(SiChannelId.DEVICE_CLASS, new UnsignedDoublewordElement(30051)), //
-						m(SiChannelId.DEVICE_TYPE, new UnsignedDoublewordElement(30053)).debug(), //
-						new DummyRegisterElement(30055, 30056), //
-						m(SiChannelId.SERIAL_NUMBER, new UnsignedDoublewordElement(30057)).debug(), //
-						m(SiChannelId.SOFTWARE_PACKAGE, new UnsignedDoublewordElement(30059))), //
-				new FC3ReadRegistersTask(30199, Priority.ONCE, //
-						m(SiChannelId.WAITING_TIME_UNTIL_FEED_IN, new UnsignedDoublewordElement(30199))), //
-				new FC3ReadRegistersTask(30201, Priority.LOW, //
+//				new FC3ReadRegistersTask(30051, Priority.ONCE, //
+//						m(SiChannelId.DEVICE_CLASS, new UnsignedDoublewordElement(30051)), //
+//						m(SiChannelId.DEVICE_TYPE, new UnsignedDoublewordElement(30053)).debug(), //
+//						new DummyRegisterElement(30055, 30056), //
+//						m(SiChannelId.SERIAL_NUMBER, new UnsignedDoublewordElement(30057)).debug(), //
+//						m(SiChannelId.SOFTWARE_PACKAGE, new UnsignedDoublewordElement(30059))), //
+//				new FC3ReadRegistersTask(30199, Priority.ONCE, //
+//						m(SiChannelId.WAITING_TIME_UNTIL_FEED_IN, new UnsignedDoublewordElement(30199))), //
+				new FC3ReadRegistersTask(30201, Priority.HIGH, //
```

Comment:
This change does not really seem to belong in this pull-request - but it's ok. Just if we don't those registers anymore, please actually remove them. Otherwise add a comment on why they are kept commented.

Reply:
This was a try purpose on a real system, SMA had some read register mistakes, I ll reverse it back